content/english/anti-vegan/no-food-is-fully-vegan.md

```diff
@@ -11,33 +11,104 @@ weight: 5
+> - The Intentional Killing of Field Animals and Ethical Veganism *by*  Joe Wills
+> 
+> (Argument #3) "While not logically impossible, it is *almost impossible* to avoid being complicit in suffering with our current food chain. Veganism is a way of living which seeks to exclude, *as far as is possible and practicable*, all forms of exploitation of, and cruelty to, animals [...]. Raising an animal from forcible impregnation to their untimely death merely to satisfy our desire for taste is not comparable to deaths we have little way of avoiding as consumers."  
 > - Definition of veganism verbatim from [The Vegan Society](https://www.vegansociety.com/go-vegan/definition-veganism)
 
 ## Detail - Analytic
 
-### Intention vs Accident
+### Argument #1 *Moral Consistency*
+
+Trying to trip vegans up by exposing real or imagined inconsistencies is common rhetoric. The reality is that vegans (being human) are typically not perfect. Vegans are subject to the constraints of time, money, knowledge, availability of resources, and a million other factors. 
+
+That a vegan may fail in no way invalidates the moral claim that the exploitation of animals is wrong.
+
+> Argument #1 is a fallacy (the appeal to hypocrisy).
+
+### Argument #2 *Intent & Knowledge*
+
+Vegans must eat and, where there are limited options, follow the path of least harm.
+
+
+>A golden rule of moral philosophy is, "all things being equal you ought to choose the action that does the least harm”.
+
+While it is true that some animals do die in the production and harvesting of crops vegans do not ‘accept these deaths’ without question.  Where a crop farmer fails to *exercise due diligence to minimize the harm* and deaths occur, an obligation falls on vegans *as far as is possible and practicable* to seek alternatives (e.g. moving towards organic vegetables, buying from vegan farms, avoiding mono-cultures etc.)
+
+> ### Doctrine of Double Effect
+> - the nature of the act is itself good, or at least morally neutral;  
+> - the individual intends the good effect and does not intend the bad effect, either as a means to the good or as an end in itself;  
+> - the good effect outweighs the bad effect in circumstances sufficiently grave to justify causing the bad effect, and the individual exercises due diligence to minimize the harm.  
+> 
+> *from* Double-Effect Reasoning: Doing Good and Avoiding Evil *by* T. A. Cavanaugh.
+ 
+While cattle farmer's *intend* to kill the cattle they raise, the crop farmer's intention is primarily to grow crops. Any deaths that may occur during crop harvest are a side-effect of a morally neutral act. 
+
+- A counter might follow that 'regardless of intention or acceptance the animals die, so I might as well get my meat.'
+
+This counter argument fails to take into account an individual's foreknowledge of what will occur.
+
+* A cattle farmer has foreknowledge that the cattle will be slaughtered and intends for them to be slaughtered.
```

Comment:
Can we substitute the word `has foreknowledge` with `knows without a doubt` or just `knows`?

Reply:
Left first foreknowledge.

Changed seconds to: 

A cattle farmer knows that the cattle will be slaughtered and intends for them to be slaughtered.

and

A crop farmer knows that some animals (e.g rabbits, birds, rodents) may be killed...

Reply:
Works for me!